packages/core/lib/when.js

```diff
@@ -0,0 +1,21 @@
+const { assertConfigSetters } = require('./configSetters')
+const group = require('./group')
+
+module.exports = when
+
+/**
+ * Applies an array of webpack blocks only if `condition` is true.
+ *
+ * @param {boolean} condition         Condition to test
+ * @param {Function[]} configSetters  Array of functions as returned by webpack blocks.
+ * @return {Function}
+ */
+function when (condition, configSetters) {
+  assertConfigSetters(configSetters)
+
+  if (condition) {
```

Comment:
Would be nice to have a check above that `condition` is a boolean.

Reply:
I had intentionally *not* done this, on the logic that `condition` could be any truthy or falsy value...

Reply:
I'm fine with `if (condition)`

Reply:
I am for consistency between code and docs. If we allow any values and cast them to booleans then docs should say that.